var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -45,10 +45,15 @@ class Gtkplus(AutotoolsPackage):
     depends_on("pango~X", when='~X')
     depends_on("pango+X", when='+X')
     depends_on('gobject-introspection', when='+X')
+    depends_on('shared-mime-info')
 
     patch('no-demos.patch')
 
     def patch(self):
         # remove disable deprecated flag.
         filter_file(r'CFLAGS="-DGDK_PIXBUF_DISABLE_DEPRECATED $CFLAGS"',
                     '', 'configure', string=True)
+
+    def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
+        spack_env.prepend_path("XDG_DATA_DIRS",
+                               self.prefix.share)
```

Comment:
Hi @adamjstewart , I just followed what was being done in other packages: atk, gdk-pixbuf and pango. I think all of these packages need `XDG_DATA_DIRS` at build and run-time. Should we update all of the packages to be consistent?